rules/windows/builtin/security/win_security_susp_possible_shadow_credentials_added.yml

```diff
@@ -0,0 +1,25 @@
+title: Possible Shadow Credentials Added
+id: f598ea0c-c25a-4f72-a219-50c44411c791
+status: experimental
+description: Detects possible addition of shadow credentials to an active directory object.
+author: Nasreddine Bencherchali (rule), Elastic (idea)
+references:
+    - https://www.elastic.co/guide/en/security/7.17/prebuilt-rule-0-16-1-potential-shadow-credentials-added-to-ad-object.html
```

Comment:
```suggestion
    - https://www.elastic.co/guide/en/security/master/potential-shadow-credentials-added-to-ad-object.html
```

Reply:
I don't like to ref master. Because if future changes apply to the rule the ref won't match the logic of the rule at the time of writing.

Thanks

Reply:
I don't see a lot to change from the original logic. and here is the current one (8.4): https://www.elastic.co/guide/en/security/8.4/potential-shadow-credentials-added-to-ad-object.html